scripts/functions.py

```diff
@@ -0,0 +1,8 @@
+def opts2dmpairArgs(opt: dict, seed: int) -> dict:
```

Comment:
rename to `utils.py` as more conventional

Reply:
Done